src/styles/style.scss

```diff
@@ -0,0 +1,115 @@
+$Secondary: #616070;
```

Comment:
Goog job! You can write scss variables in another file like 'variables.scss' and import them in this file. Also I think use '*' it's not good idea, maybe you have another ideas about this?